schemas/expconf/v0/s3.json

```diff
@@ -42,6 +42,14 @@
             ],
             "default": null
         },
+        "prefix": {
```

Comment:
can you add the prefix field to the `s3 checkpoint storage (valid)` test case in `schemas/test_cases/v0/unions.yaml`?